### PR TITLE
Add endpoints for my assets

### PR DESCRIPTION
**Pull request by andrejridzik**
Adds integration with My Library

### CODE REVIEW

**Review by marcel-vesely-kinit — COMMENTED** [review]
The integration with My Library works just fine, but I don't know whether I'm allowed to approve this request as I haven't checked changes made to the tests and I don't plan to (as I've already forgotten how they work and so I don't want to waste time on it as of right now).

 I suppose I will leave it to @mtkinit to check it instead :)

File: backend/app/routers/aiod.py [hunk 1]
```diff
@@ -32,6 +32,13 @@ async def get_datasets(pagination: Pagination = Depends()) -> Any:
     return await get_assets(asset_type=AssetType.DATASETS, pagination=pagination)
 
 
+@router.get("/datasets/my", response_model=list[Dataset])
```
Comment (marcel-vesely-kinit) [inline backend/app/routers/aiod.py:35]: These my functions should also take pagination parameters, shouldnt it

Comment (andrejridzik) [reply backend/app/routers/aiod.py:35]: You are right.. I did not add it there as it is not supported by MyLibrary at the moment. But I can actually try to mimic it and create an GitHub issue (feature request) for it. 👍

Comment (andrejridzik) [reply backend/app/routers/aiod.py:35]: @marcel-vesely-kinit Pagination added, please have a look.

**Comment by andrejridzik** [comment]
> The integration with My Library works just fine, but I don't know whether I'm allowed to approve this request as I haven't checked changes made to the tests and I don't plan to (as I've already forgotten how they work and so I don't want to waste time on it as of right now).
> 
> I suppose I will leave it to @mtkinit to check it instead :)

There are actually not that many changes to the tests made 😄 . I think it would be a good practice to at least go through the changes. The automated tests are running as part of CI, so at least we can be sure they are passing 😉 .

**Review by marcel-vesely-kinit — APPROVED** [review]

**Review by mtkinit — APPROVED** [review]

**Pull request merged**